recipes/dcgp-python/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.3" %}
+
+package:
+  name: dcgp-python
+  version: {{ version }}
+
+source:
+  url: https://github.com/darioizzo/dcgp/archive/v{{ version }}.tar.gz
+  sha256: 85c5318cdb514dcd3fc91ba45b9f112bdd4c0a321c311b7b6bd115cf7d03f294
+
+build:
+  number: 0
+  skip: true  # [(win and py<35)]
```

Comment:
Can you try `win or py<35`

Reply:
Won't that skip Windows builds altogether?

Reply:
Yes, I thought that was the intention. Or maybe try just py<35 to fix the build issues.

Reply:
Ah no, the idea was to have Windows builds only for Python 3 (because for Python 2 the compiler is too old).

Reply:
But the package conflicts you get currently seem to be limited to Python 2.7.

Reply:
What selector syntax should I use to disable Python 2.x build on Windows?

Also, are the info in this page still relevant?

https://github.com/conda-forge/staged-recipes/wiki/VC-features

Reply:
My suggestion is to disable python 2.7 for all systems, just for a test to see if this resolves the issues with the package conflict.

Reply:
Ok, I'll try. Thanks for the hint.

Reply:
Looks like the conflicts are still there...